### DISCUSSION TITLE
WebServer in HomeSpan

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [HomeSpan/HomeSpan#214] [Ideas]
Maybe we can add a web server to homespan so that we can change some variables from the www level and not the code itself?

As far as I know, the web server is already implemented, so after connecting to our wifi it could change into an interface for changing variables that cannot be edited from the home level

**Comment by HomeSpan** [comment]
HomeSpan supports the inclusion of a second web server (running in parallel to the HomeKit server).  See this [FAQ](https://github.com/HomeSpan/HomeSpan/blob/master/docs/FAQ.md#can-you-add-a-web-server-to-homespan) for details and references.
